src/components/MoviesList/MoviesList.tsx

```diff
@@ -1,6 +1,21 @@
-import React from 'react';
+import MovieCard from '../MovieCard';
+import { Movie } from '../../types/movie';
 import './MoviesList.scss';
 
-export const MoviesList: React.FC = () => (
-  <>Put the list here</>
+type Props = {
+  movies: Movie[];
+};
+
+export const MoviesList: React.FC<Props> = ({ movies }) => (
+  <div className="movies">
+    {movies.map(movie => (
+      <MovieCard
+        key={movie.imdbId}
+        title={movie.title}
+        description={movie.description}
+        imgUrl={movie.imgUrl}
+        imdbUrl={movie.imdbUrl}
```

Comment:
you should pass here the whole `movie` object as props, not each part separately.